### ISSUE TITLE
Keyboard navigation on on Logging view doesn't work as expected

### ISSUE THREAD

**Issue by DanTup** [flutter/devtools#363]
If you click on a row with the mouse, then try to use the keyboard, you seem to end up on the first, last, or some random row.

**Comment by kenzieschmoll** [comment]
Fixed as of Devtools 0.1.4
